tests/simple_server.py

```diff
@@ -73,6 +65,12 @@ def log_request(self, code='-', size='-'):
 # Allow re-use so you can re-run tests as often as you want even if the
 # tests re-use ports. Otherwise TCP TIME-WAIT prevents reuse for ~1 minute
 six.moves.socketserver.TCPServer.allow_reuse_address = True
-httpd = six.moves.socketserver.TCPServer(('', PORT), handler)
 
-httpd.serve_forever()
+try:
+  httpd = six.moves.socketserver.TCPServer(('localhost', 0), handler)
+  port_message = 'bind succeeded, server port is: ' \
+      + str(httpd.server_address[1])
+  print(port_message)
+  httpd.serve_forever()
+except:
+  print("bind failed")
```

Comment:
we discussed this and came to the conclusion that the exception handling is not useful in this version -- the log message  is not needed and actively hides possible other error (exception) messages

Reply:
this applies to all servers